docfx_project/articles/golang-aha/article.md

```diff
@@ -0,0 +1,508 @@
+# Golang Aha! Moments
+
+## Introduction
+As we (the OpenZiti team) has progressed on our Go journey, we've stumbed on various
+obstacles, settled on some best practices and hopefully gotten better at writing Go
+code. This document is share some of the stumbling blocks and solutions, both for
```

Comment:
"document is meant to share"? it's just missing a certain something/word :)

Reply:
Fixed